README.md

```diff
@@ -58,6 +58,8 @@ collect.global_variables                               | 5.1           | Collect
 collect.info_schema.clientstats                        | 5.5           | If running with userstat=1, set to true to collect client statistics.
 collect.info_schema.innodb_metrics                     | 5.6           | Collect metrics from information_schema.innodb_metrics.
 collect.info_schema.innodb_tablespaces                 | 5.7           | Collect metrics from information_schema.innodb_sys_tablespaces.
+collect.info_schema.innodb_cmp                	       | 5.5           | Collect metrics from information_schema.innodb_cmp.
+collect.info_schema.innodb_cmpmem              	       | 5.5           | Collect metrics from information_schema.innodb_cmpmem.
```

Comment:
The spacing seems off here.

Reply:
Maybe let's give this a nicer description, like `Collect InnoDB buffer pool compression metrics from information_schema.innodb_cmpmem`.